doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -177,7 +177,9 @@ int main(int argc, char **argv)
   // We will reuse the old goal that we had and plan to it.
   // Note that this will only work if the current state already
   // satisfies the path constraints. So, we need to set the start
-  // state to a new pose.
+  // state to a new pose. We also increase the default planning time from 5s to 10s.
+  
+  my_plan.planning_time_ = 10;
```

Comment:
I'm pretty sure this does not work at all.
The `plan` method does not read this value, but overrides it with the [*actual* time used by the planner](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp#L705)

You should use `group.setPlanningTime(10.0)` instead.